be/src/exec/hash_joiner.h

```diff
@@ -118,6 +119,22 @@ struct HashJoinerParam {
     const TJoinDistributionMode::type _distribution_mode;
 };
 
+inline bool is_left_semi_join_or_inner_join(TJoinOp::type join_type) {
+    return join_type == TJoinOp::LEFT_SEMI_JOIN || join_type == TJoinOp::INNER_JOIN;
+}
+
+inline bool has_post_probe(TJoinOp::type join_type) {
+    return join_type == TJoinOp::RIGHT_OUTER_JOIN || join_type == TJoinOp::RIGHT_ANTI_JOIN ||
+           join_type == TJoinOp::FULL_OUTER_JOIN;
+}
+
+inline bool is_spillable(TJoinOp::type join_type) {
```

Comment:
at present, all join types support spill, so not need to use this function.

Reply:
at this time, null aware left anti join has not support. we will implement it later.